apps/src/javalab/JavalabView.jsx

```diff
@@ -114,7 +116,35 @@ class JavalabView extends React.Component {
     this.props.appendOutputLog('Compiled!');
   };
 
+  renderSettings = () => {
+    const {isDarkMode} = this.props;
+    return [
+      <a onClick={this.props.toggleDarkMode}>
+        Switch to {isDarkMode ? 'light mode' : 'dark mode'}
+      </a>
+    ];
+  };
+
+  getButtonStyles = isSettings => {
+    const {isDarkMode} = this.props;
+    if (isDarkMode) {
```

Comment:
Suggestion: Change this to `if ... else if ... else` - That will prevent future maintainers from accidentally inserting statements between the three conditions.